src/pages/EditOrderPage.js

```diff
@@ -7,7 +7,7 @@ import { useNavigate, useParams } from 'react-router-dom';
 import { deleteOrder, updateOrder } from '../api';
 import { AuthContext } from '../context/auth.context';
 import { deleteShopOrder } from '../redux/features/orders/ordersSlice';
-import { addToCart, clearCart, setItemAmount, handleAddedDeliveryFee } from '../redux/features/items/itemsSlice';
+import { addToCart, clearCart, setItemAmount, handlefreeDelivery } from '../redux/features/items/itemsSlice';
```

Comment:
`handlefreeDelivery` 
This should be camel case like this: `handleFreeDelivery`